common/models/src/utils.rs

```diff
@@ -91,3 +91,7 @@ impl SeqIdGenerator {
             .fetch_add(1, std::sync::atomic::Ordering::Relaxed)
     }
 }
+
+pub fn splice_address(address: String, port: u32) -> String {
```

Comment:
This function just create one address, so it's better to use `build_address`.

Reply:
ok.

Reply:
This function does not need the ownership of `address`.

#### Example 

```rust
pub fn build_address(address: &str, port: u16) -> String {
    xxxxxxxx
}
```